backend/benefit/calculator/api/v1/serializers.py

```diff
@@ -272,6 +273,22 @@ def validate(self, data):
                 raise serializers.ValidationError(
                     {"end_date": _("End date cannot be empty")}
                 )
+            state_aid_max_input = self.context["request"].data["calculation"][
+                "state_aid_max_percentage"
+            ]
+            if (
```

Comment:
Could be written:

```
if (
    not state_aid_max_input or state_aid_max_input
    not in [choice[0] for choice in STATE_AID_MAX_PERCENTAGE_CHOICES]
):
    ...
```

or if the `STATE_AID_MAX_PERCENTAGE_CHOICES`  were defined as a Django enum, like something like this:
```
class STATE_AID_MAX_PERCENTAGE_CHOICES(models.IntegerChoices):
    FIFTY = 50, "50%"
    HUNDRED = 100, "100%"
```

then there would be attributes `.values` and `.choices` available and the statement could be written:
```
if not state_aid_max_input or state_aid_max_input not in STATE_AID_MAX_PERCENTAGE_CHOICES.values:
    ...
```
but… don't know if it's really worth the trouble.